src/components/Safety/AnonymousSafetyReportModal.tsx

```
@@ -0,0 +1,218 @@
+      severity,
+      description,
+      is_anonymous: isAnonymous,
+    });
+
+    setIsSubmitting(false);
+    if (success) {
+      setIsSuccess(true);
+      setTimeout(() => {
+        setIsSuccess(false);
+        onClose();
+        setDescription('');
```

ESLint

[error] Replace `''` with `""`

(prettier/prettier)